adaptivefiltering/execute.py

```diff
@@ -66,14 +66,32 @@ def apply_adaptive_pipeline(
 
     # We decrease the logging level
     logger.setLevel(logging.INFO)
-
     # Ensure existence of output directory
+    # This is not written in the log file
```

Comment:
Should be?

Reply:
Changed it to log, that the dict was created.